packages/commonwealth/client/scripts/views/pages/notification_settings/index.tsx

```diff
@@ -498,6 +504,91 @@ const NotificationSettingsPage = () => {
             </div>
           );
         })}
+      <div style={{ marginTop: 26 }}>
+        <CWText
+          type="h4"
+          fontWeight="semiBold"
+          className="chain-events-section-margin"
+        >
+          Snapshot Subscriptions
+        </CWText>
+        <div className="column-header-row">
+          <CWText
+            type={isWindowExtraSmall(window.innerWidth) ? 'caption' : 'h5'}
+            fontWeight="medium"
+            className="column-header-text"
+          >
+            Community
+          </CWText>
+          <CWText
+            type={isWindowExtraSmall(window.innerWidth) ? 'caption' : 'h5'}
+            fontWeight="medium"
+            className="column-header-text"
+          >
+            Email
+          </CWText>
+          <CWText
+            type={isWindowExtraSmall(window.innerWidth) ? 'caption' : 'h5'}
+            fontWeight="medium"
+            className="last-column-header-text"
+          >
+            In-App
+          </CWText>
+        </div>
+      </div>
+      {Object.entries(bundledSnapshotSubs)
+        .sort((x, y) => x[0].localeCompare(y[0]))
+        .map(([chainName, subs]) => {
+          const getChainName = () => {
+            const getCurrentChain = subs.find((chain) => {
+              if (Number(chain.id) === Number(chainName)) {
+                return chain.snapshotId;
+              }
+            });
+
+            const chainNameUrl = getCurrentChain.snapshotId.replace('.eth', '');
+
+            if (chainNameUrl === 'dydxgov') {
```

Comment:
Why this hardcoded portion? Snapshot spaces can have a huge variety of names so we should avoid hardcoding anything here.